### PR TITLE
SAK-44836 Profile: Recorded Name Pronunciation audio

### PR DESCRIPTION
**Pull request by MRutea**

### CODE REVIEW

File: profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyNamePronunciationEdit.html [hunk 1]
```diff
@@ -170,6 +170,7 @@
             audio.muted = true;
             audio.srcObject = microphone;
             var options = {
+                sampleRate: 43100,
```
Comment (ottenhoff) [inline profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyNamePronunciationEdit.html:173]: this seems odd

Comment (mpellicer) [reply profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyNamePronunciationEdit.html:173]: I remember this was reported to me by NYU several years ago, they mentioned the pitch issue and I remember changing this parameter, maybe I only changed the param in their branch.

This sample rate doesn't look very standard to me, maybe we should fit to an standard one?

Comment (ottenhoff) [reply profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyNamePronunciationEdit.html:173]: yeah i think 44100 is the standard?

Comment (MRutea) [reply profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyNamePronunciationEdit.html]: Yes, it seems that 44100 is the standard, but in my testing it seems a little high yet.

Changed to 44100.

Tested in Chrome, Edge, Firefox and Opera.

Comment (mpellicer) [reply profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyNamePronunciationEdit.html]: After setting to 44100, how does your voice sound?

Comment (MRutea) [reply profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyNamePronunciationEdit.html]: @mpellicer 
After setting to 44100 my voice still sounds natural. I think that there's little difference between the trunk and 44100, just a bit, but enough. In short, now there is no high-pitched feel. And I think that's the important question.

**Review by mpellicer — APPROVED** [review]
Looks tested any other thoughts Sam? @ottenhoff

**Pull request merged**